web/player.ts

```diff
@@ -277,107 +286,11 @@ export default class Player {
     this.sendHandleEvent(PlayerEventEnum.PlayerReady, this)
   }
 
-  handleFrameData(messages) {
-    // console.log(`received frames ${messages[0].keyframeNumber} - ${messages[messages.length-1].keyframeNumber}`)
-    for (const frameData of messages) {
-      let geometry = new BufferGeometry();
-      geometry.setIndex(
-        new Uint32BufferAttribute(frameData.bufferGeometry.index, 1)
-      );
-      geometry.setAttribute(
-        'position',
-        new Float32BufferAttribute(frameData.bufferGeometry.position, 3)
-      );
-      geometry.setAttribute(
-        'uv',
-        new Float32BufferAttribute(frameData.bufferGeometry.uv, 2)
-      );
-
-      this.meshBuffer.set(frameData.keyframeNumber, geometry );
-    }
-
-    if (typeof this.onMeshBuffering === "function") {
-      const minimumBufferLength = this.targetFramesToRequest * 2;
-      // console.log('buffering ', this.meshBuffer.size / minimumBufferLength,',  have: ', this.meshBuffer.size, ', need: ', minimumBufferLength )
-      this.onMeshBuffering(this.meshBuffer.size / minimumBufferLength);
-    }
-  }
-
-  getCurrentFrameNumber():number {
-    const encoderWindowWidth = this.encoderWindowSize * this.encoderByteLength;
```

Comment:
We shouldn't be deleting all this, still need it for some browsers (e.g., Firefox)